src/stores/index.ts

```diff
@@ -92,6 +78,6 @@ export function useState(): {
 
       window.erc725Account = undefined
     },
-    recalcTokens,
+    recalcTokens: recalculateAssets,
```

Comment:
<img width="433" alt="Screenshot 2023-11-24 at 12 48 41" src="https://github.com/lukso-network/universalprofile-test-dapp/assets/36865532/08c61211-b8ef-4437-be9b-b9c9133882db">

Suggest renaming `recalcTokens` to `recalculateAssets`:

```suggestion
    recalculateAssets,
```

Reply:
Makes sense, I've made symbolic replace and I guess it doesn't pick this up.